tests/Tests.Configuration/Tests.Configuration.csproj

```diff
@@ -3,6 +3,6 @@
     <TargetFramework>netstandard2.0</TargetFramework>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="OpenSearch.OpenSearch.Managed" Version="0.1.0-canary.0.17" />
+    <PackageReference Include="OpenSearch.OpenSearch.Managed" Version="0.1.0-canary.0.32" />
   </ItemGroup>
 </Project>
```

Comment:
Another missing new line? (I realize it was already missing)

Reply:
Updated in 149f3a7534df2ab066ac6e79a79588a698ab4091